assets/app/views/builds.html

```diff
@@ -100,7 +100,7 @@
                         <span ng-if="build.spec.source.type == 'None'">
                           <i>none</i>
                         </span>
-                        <span ng-if="build.spec.source.type == 'Git'" ng-bind-html='build.spec.source.git.uri | githubLink : build.spec.source.git.ref : build.spec.source.contextDir | linky'></span>
+                        <span class="word-break" ng-if="build.spec.source.type == 'Git'" ng-bind-html='build.spec.source.git.uri | githubLink : build.spec.source.git.ref : build.spec.source.contextDir | linky'></span>
```

Comment:
@jhadvig the reason you aren't seeing it in your example is because it is breaking at the '/'
If it was something like the following it wouldn't wrap.

https://github.com/thisisalongnonbreakingstringthisisalongnonbreakingstringthisisalongnonbreakingstringthisisalongnonbreakingstringthisisalongnonbreakingstringthisisalongnonbreakingstringthisisalongnonbreakingstringthisisalongnonbreakingstring

And in order to break that in FF and IE we would need to use ```class="word-break-all"```

<img width="1063" alt="screen shot 2016-05-06 at 11 58 40 am" src="https://cloud.githubusercontent.com/assets/1874151/15078744/ad5ef2ba-1382-11e6-8dc7-2b45b7eb8f72.png">